src/app/introduction/activity-2/activity-2.component.ts

```diff
@@ -15,17 +16,26 @@ export class PcpolicyComponent implements OnInit {
     language: any;
     email: String;
     private _status: object = {stage: 1, activity: 2};
-    public completed = true;
```

Comment:
How was this true till now..

Reply:
@paavininanda this is Peace Corps Policy component. So, the user just sees it and the level is completed.